### PR TITLE
Navbar now collapsable 

### PR DESCRIPTION
**Pull request by mjrouser**
Added Bootstrap CDNs for CSS/JS and required JQuery.  These versions are ahead of what was in the 'lib' folder.  

Created collapsable navbar with toggle button.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -6,24 +6,60 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Generic 'find-a' app for geographic datasets">
     <meta name="author" content="Code for Boston">
+
+
+    <!-- CDNs for Bootstrap stylesheets-->
+    <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css" type="text/css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css" type="text/css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap-theme.min.css">
```
Comment (paulswartz) [inline index.html]: Which Bootstrap theme is this?  I didn't notice anything visually different on your fork.  

If it's not affecting anything, then you shouldn't need to include any new CSS, as we're already including the regular Bootstrap CSS.

File: index.html [hunk 2]
```diff
@@ -6,24 +6,60 @@
...
     <![endif]-->
+
+
   </head>
 
   <body>
+
+  <!-- External links for the Jquery and JS needed for Bootstrap -->
+  <script src="//ajax.googleapis.com/ajax/libs/jquery/1.11.0/jquery.min.js"></script>
+  <script src="//netdna.bootstrapcdn.com/bootstrap/3.1.1/js/bootstrap.min.js"></script>
```
Comment (paulswartz) [inline index.html]: All the Javascript gets loaded via the `src/script.js` file.  If you take a look at #59, you can see how to include a new file.

File: index.html [hunk 3]
```diff
@@ -6,24 +6,60 @@
...
+           <span class="icon-bar"></span>
+           <span class="icon-bar"></span>
+         </button>
+         <a class="navbar-brand" href="/">Finda</a>
+        </div> 
+
+        <!--Collect the nav links and content for toggling-->
+        <div class="collapse navbar-collapse" id="finda-navbar-collapse">  
+          <form id="search" class="navbar-form navbar-left" role="search">
+            <div class="form-group">
```
Comment (paulswartz) [inline index.html:38]: The collapsing work great, but seems to have make the search buttons a bit wonky.

**Comment by mjrouser** [comment]
1. I put the theme CSS in as a just-in-case, but I'll take it out
2. The search buttons were not wonky when I was using the CDNs on local, now that I have taken them out and am trying to rely on linked CSS/JS I see what you are talking about.  I am not sure if there is a conflict between the custom CSS and the bootstrap file or what. Need to dig in a bit more

**Comment by mjrouser** [comment]
Wonky-ness is fixed

**Comment by mjrouser** [comment]
Tried to add bootstrap.min to src/script.js.  Not sure if I executed it properly

**Comment by paulswartz** [comment]
Now that @ohnorobo merged #59, all you should need to do for this branch is update `index.html`.  If you don't get around to it today, we can take a look at it tonight at the hack night.

File: src/script.js [hunk 4]
```diff
@@ -2,6 +2,7 @@ require.config({
   baseUrl: 'src/',
   paths: {
     'jquery': '../lib/jquery-1.10.2',
+    'bootstrap': '../lib/bootstrap.min',
```
Comment (paulswartz) [inline src/script.js:5]: You can back out this line too, since it was added by an earlier pull request.

File: index.html [hunk 5]
```diff
@@ -6,12 +6,60 @@
...
+
+    <!-- CDNs for Bootstrap stylesheets
+    <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css" type="text/css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css" type="text/css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap-theme.min.css">
+    -->
+
+    <!-- Custom CSS -->
     <link rel="stylesheet" type="text/css" href="styles/style.css" />
+<<<<<<< HEAD
```
Comment (paulswartz) [inline index.html]: It looks like you checked in the merge conflicts.  You might have made the changes locally, but forgotten to `git add` them?

File: index.html [hunk 6]
```diff
@@ -6,12 +6,60 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <meta name="description" content="Generic 'find-a' app for geographic datasets">
     <meta name="author" content="Code for Boston">
+
+
+    <!-- CDNs for Bootstrap stylesheets
+    <link rel="stylesheet" href="//ajax.googleapis.com/ajax/libs/jqueryui/1.10.4/themes/smoothness/jquery-ui.css" type="text/css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap.min.css" type="text/css">
+    <link rel="stylesheet" href="//netdna.bootstrapcdn.com/bootstrap/3.1.1/css/bootstrap-theme.min.css">
+    -->
```
Comment (paulswartz) [inline index.html]: These are already included as well by the existing CSS.

**Comment by ohnorobo** [comment]
we lost the About link.

**Comment by paulswartz** [comment]
Yeah, just noticed that. I'm taking a look.

**Pull request merged**